hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/mover/package-info.java

```diff
@@ -0,0 +1,27 @@
+/**
```

Comment:
Do we need to add this file?

Reply:
It is to pass the checkstyle as I added a new file to this package without package-info..